README.md

```diff
@@ -31,21 +31,14 @@ Download the Python 3.8 gtsam wheel for Mac [here](https://github.com/borglab/gt
 ```bash
 pip install ~/Downloads/gtsam-4.1.1-py3-none-any.whl
 ```
-Now, inside the environment, build pydegensac:
```

Comment:
thanks for the PR @womackj1 . Do you mind adding these instead at the end of the readme, maybe as "Compiling Additional Verifiers"?

Reply:
Of course! The readme has the new section "Compiling Additional Verifiers".